src/app/services/evt-model.service.ts

```diff
@@ -80,15 +82,34 @@ export class EVTModelService {
     shareReplay(1),
   );
 
+  // WITNESSES
+  public readonly witnessesData$ = this.editionSource$.pipe(
+    map((source) => this.witnessesParser.parseWitnessesData(source)),
+    shareReplay(1),
+  );
+
+  public readonly witnesses$ = this.witnessesData$.pipe(
```

Comment:
Which is the difference between witnesses and witnessesData? I think we should understand it somehow from the name

Reply:
I think we can delete witnessesData$